x-pack/elastic-agent/docs/elastic-agent-command-line.asciidoc

```diff
@@ -176,6 +184,112 @@ elastic-agent inspect output --output default --program filebeat
 <hr>
 ++++
 
+[discrete]
+[[elastic-agent-install-command]]
+== elastic-agent install
+
+Install {agent} permanently on the system and manage it by using the system's
+service manager. The agent will start automatically after installation is
+complete.
+
+We recommend that you run this command as the root user because some
```

Comment:
This is required to run as root, it is not recommended. The command will not run unless they are root. They must be root as well, or writing the files in there correct locations will fail.

Also I think we should highlight "Administrator" as well for the Windows case.